composer.json

```diff
@@ -36,7 +36,7 @@
     "require-dev": {
         "infection/infection": "^0.26.6",
         "laminas/laminas-coding-standard": "~2.3.0",
-        "maglnet/composer-require-checker": "^3.8.0",
+        "maglnet/composer-require-checker": "^4.0.0",
```

Comment:
This package requires PHP `^8.0` to function: https://github.com/maglnet/ComposerRequireChecker/blob/baa11a4e9e5072117e3d180ef16c07036cafa4a2/composer.json#L29

I think our safest bet would be to remove it from here, and let our CI setup deal with this sort of upgrade perhaps?